curriculum/challenges/english/25-front-end-development/lecture-working-with-client-side-storage-and-crud-operations/6733ff4a9319c8486750886c.md

```diff
@@ -2,13 +2,17 @@
 id: 6733ff4a9319c8486750886c
 title: What Is localStorage, and What Are Some Common Methods?
 challengeType: 19
-# videoId: nVAaxZ34khk
+videoId: H-FAQKeCtlY
```

Comment:
Video has errors in it. So this PR will remain in draft status until I can get the correct video.